src/rpc/blockchain.cpp

```diff
@@ -1754,8 +1754,10 @@ static UniValue getblockstats(const JSONRPCRequest& request)
             "}\n"
                 },
                 RPCExamples{
-                    HelpExampleCli("getblockstats", "1000 '[\"minfeerate\",\"avgfeerate\"]'")
-            + HelpExampleRpc("getblockstats", "1000 '[\"minfeerate\",\"avgfeerate\"]'")
+                    HelpExampleCli("getblockstats", R"('"00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09"' '["minfeerate","avgfeerate"]')") +
+                    HelpExampleCli("getblockstats", R"("1000 '["minfeerate","avgfeerate"]'")") +
+                    HelpExampleRpc("getblockstats", R"("00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09", ["minfeerate","avgfeerate"])") +
+                    HelpExampleRpc("getblockstats", R"("1000, ["minfeerate","avgfeerate"]")")
```

Comment:
Same problem as above here, there are two extra quotation marks that break the example:
```
$ curl --user __cookie__ --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "getblockstats", "params": ["1000, ["minfeerate","avgfeerate"]"] }' -H 'content-type: text/plain;' http://127.0.0.1:8332/
Enter host password for user '__cookie__':
{"result":null,"error":{"code":-32700,"message":"Parse error"},"id":null}
```

Reply:
```suggestion
                    HelpExampleRpc("getblockstats", R"(1000, ["minfeerate","avgfeerate"])")
```

Reply:
Thanks for catching this. I fixed and rebased on latest master and added you as co-author on the commit.